src/main/kotlin/calculator/Operation.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+import java.lang.IllegalArgumentException
+import java.util.function.BiFunction
+
+enum class Operation(val char: String, val operate: BiFunction<Int, Int, Int>) {
```

Comment:
코틀린은 함수를 일반 값처럼 다룰 수 있습니다. 이를 일급 시민(first-class citizen)인 함수라고 해요. `BiFunction` 대신 `(Int, Int) -> Int`를 사용하면 어떨까요?

```suggestion
enum class Operation(val char: String, val operate: (Int, Int) -> Int) {
```